StepDefinitions/TestSteps.cs

```diff
@@ -0,0 +1,131 @@
+using System;
+using TechTalk.SpecFlow;
+
+namespace TestSpecs {
+    [Binding]
+    class TestStpes
+    {
+        [Given(@"I am on the (.*) website")]
+        public void GivenIAmOnTheNLWebsite(string countryCode)
+        {
+            setCountryCode(countryCode);
+            Driver.Navigate().GoToUrl(baseURLCK);
+        }
+
+        [Given(@"All cookies are set")]
+        public void GivenAllCookiesAreSet()
+        {
+            Driver.Manage().Cookies.DeleteCookieNamed("PVH_COOKIES_GDPR");
```

Comment:
* You could create a `CookiesManager` and put it in `TestContext`
* Also you could put these cookies name as **const**, as the value of `const` has been setup when compiling, and `readonly` is in runtime, we want to save time :)